### ISSUE TITLE
The root CA is not trusted...

### ISSUE THREAD

**Issue by Robert-M-Muench** [sqlitebrowser/sqlitebrowser#2482]
Trying to connect to dbhub.io from OSX I get:

![image](https://user-images.githubusercontent.com/1826391/99407256-2cde6800-28ef-11eb-91df-8e812881c985.png)

Any idea how to fix this?

**Comment by lucydodo** [comment]
Thanks for reporting. 😄 
However, if there is detailed information that can reproduce the error, it will be more helpful in solving the problem.

**Comment by Robert-M-Muench** [comment]
Well, I just select my identity on the remote tab and bang, I see this. Nothing else done.

**Comment by justinclift** [comment]
Uh oh.  Hopefully this isn't a case of me letting some certificates expire.

Looking into it now... :smile:

**Comment by justinclift** [comment]
@Robert-M-Muench By the way, which version of the application are you using?

**Comment by justinclift** [comment]
@Robert-M-Muench Would you be ok to screenshot the **Preferences** → **Remote** → **CA certificates** dialog tab?

I'm wondering if somehow your system has older (now expired) certificates.

eg this is mine:

![image](https://user-images.githubusercontent.com/406299/99461194-6f528400-2985-11eb-8935-a9e98c0e76f5.png)

One other potential (simple!) problem worth checking, is whether your system date is set correctly?

On the odd occasion I've been mucking around with my system date for testing things, certificate and HTTPS errors are pretty common. :smile:

**Comment by Robert-M-Muench** [comment]
DB Browser for SQLite Version 3.12.1

![image](https://user-images.githubusercontent.com/1826391/99541008-96727b00-29b0-11eb-8cb6-9da66afd0a75.png)

I think it's more an OSX issue where the SSL componenten checks for the CA and dbhub.io is not a trusted CA Root.

**Comment by justinclift** [comment]
Interesting.  Those are the right certificates, so it's not a "somehow using old certificates" problem. :smile:

> I think it's more an OSX issue where the SSL componenten checks for the CA and dbhub.io is not a trusted CA Root.

In theory (!) that shouldn't be the problem.  In the code which tries connecting to the server, it's passed those certificates and told to use them for validation, rather than the system ones.

That being said... clearly something isn't working how we're expecting, so you could be onto something. :wink:

I'll try things out on our macOS build server, and see if I can replicate the problem there. :smile:

**Comment by Robert-M-Muench** [comment]
> it's passed those certificates and told to use them for validation, rather than the system ones.

Yes, but Apple doesn't know and trust you guys... so you are not an officially accepted CA Root.
